app/services/listing_service/store/shape.rb

```diff
@@ -116,7 +117,9 @@ def to_tt_model_attributes(hash)
       raise ArgumentError.new("Unknown unit type #{unit_type}")
     end
 
-    model_hash.except(:units)
+    model_hash[:url] = uniq_url(hash[:url_source], hash[:community_id])
```

Comment:
Ah, ok. I thought this was the place to configure random 'opts' to tt_model_attributes. So, misunderstood the name of the function.